### ISSUE TITLE
missed and/or incorrect fields in api logs

### ISSUE THREAD

**Issue by igomura** [minio/minio#9406]
While collecting minio logs into splunk we have found what for all GetObject requests two fields api.timeToFirstByte and api.timeToResponse are always 0s. There are no other values for many days.
Also, there are no status and statusCode fields anymore.

## Expected Behavior
Expected some variations for values of api.timeToFirstByte and api.timeToResponse.
```json
{
  "api": {
    "bucket": "dev",
    "name": "GetObject",
    "object": "uploads/db/34/db/589~C60366B4-0A4D-486E-BA96-921395D010CF/guidSplunk-C60366B4-0A4D-486E-BA96-921395D010CF/bucket_info.csv",
    "status": "OK",
    "statusCode": "200",
    "timeToFirstByte": "0.001s",
    "timeToResponse": "0.001s"
  }
}
```
## Current Behavior
```json
{
  "api": {
    "bucket": "dev",
    "name": "GetObject",
    "object": "uploads/db/34/db/589~C60366B4-0A4D-486E-BA96-921395D010CF/guidSplunk-C60366B4-0A4D-486E-BA96-921395D010CF/bucket_info.csv",
    "timeToFirstByte": "0s",
    "timeToResponse": "0s"
  }
}
```

## Steps to Reproduce (for bugs)
1. Setup logging to splunk endpoint
MINIO_AUDIT_WEBHOOK_AUTH_TOKEN="<TOKEN>"
MINIO_AUDIT_WEBHOOK_ENDPOINT="http://splunkfwd.local.net:9000/services/collector/raw"

2. In splunk, search for api.name=GetObject

## Regression
It used to work as expected in RELEASE.2020-03-19T21-49-00Z and earlier

## Your Environment
* Version used (`minio version`): 2020-04-15T19:42:18Z
* Operating System and version (`uname -a`): CentOS7, Linux minio.local.net 3.10.0-1062.4.1.el7.x86_64 #1 SMP Fri Oct 18 17:15:30 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
* Splunk version: 7.3.3

**Comment by annkam** [comment]
Good afternoon, I was going to create the same issue!
The problem with missing api.status and api.statusCode is present since **RELEASE.2019-12-17T23-16-33Z** and still persists in **RELEASE.2020-04-22T00-11-12Z**
All requests except WebUpload, WebDelete and other browser-access methods are logged without status code.
You can easily check it. Run netcat to listen on a local port. For example:

`nc -l -p 30185`

 Run old release of Minio (before lots of breaking changes in December):

```
/usr/bin/docker run --name minio \
  -p 9000:9000 \
  -e "MINIO_ACCESS_KEY=testtest" \
  -e "MINIO_SECRET_KEY=testtest123" \
  -e "MINIO_LOGGER_HTTP_ENDPOINT=http://192.168.1.112:30185" \
  -e "MINIO_AUDIT_LOGGER_HTTP_ENDPOINT=http://192.168.1.112:30185" \
  -v /tmp/minio:/export \
  minio/minio:RELEASE.2019-10-12T01-39-57Z \
  server /export
```

Create a bucket and you will get the following access log:

```
{"version":"1","deploymentid":"edec2a5e-301a-4c33-acfc-218b62d4c785","time":"2020-04-22T11:11:47.476245182Z","api":{"name":"PutBucket","bucket":"test-log","status":"OK","statusCode":200,"timeToFirstByte":"0s","timeToResponse":"314.024µs"},"remotehost":"10.11.0.1","requestID":"16081FDE1CE28596","userAgent":"MinIO (linux; amd64) minio-go/v6.0.52 mc/2020-04-04T05:28:55Z","requestHeader":{"Authorization":"AWS4-HMAC-SHA256 Credential=testtest/20200422/us-east-1/s3/aws4_request, SignedHeaders=host;x-amz-content-sha256;x-amz-date, Signature=41e5c579ab657faa7fd636bfbbd387ef95ed0573cef5cd3055d04313ac116863","Content-Length":"0","User-Agent":"MinIO (linux; amd64) minio-go/v6.0.52 mc/2020-04-04T05:28:55Z","X-Amz-Content-Sha256":"UNSIGNED-PAYLOAD","X-Amz-Date":"20200422T111147Z"},"responseHeader":{"Accept-Ranges":"bytes","Content-Length":"0","Content-Security-Policy":"block-all-mixed-content","ETag":"","Location":"/test-log","Server":"MinIO/RELEASE.2019-10-12T01-39-57Z","Vary":"Origin","X-Amz-Request-Id":"16081FDE1CE28596","X-Xss-Protection":"1; mode=block"}}
```

Stop minio and upgrade:

```
 /usr/bin/docker run --rm --name minio \
   -p 9000:9000 \
   -e "MINIO_ACCESS_KEY=testtest" \
   -e "MINIO_SECRET_KEY=testtest123" \
   -e "MINIO_LOGGER_WEBHOOK_ENABLE_target1=on" \
   -e "MINIO_LOGGER_WEBHOOK_ENDPOINT_target1=http://192.168.1.112:30185" \
   -e "MINIO_AUDIT_WEBHOOK_ENABLE_target1=on" \
   -e "MINIO_AUDIT_WEBHOOK_ENDPOINT_target1=http://192.168.1.112:30185" \
   -v /tmp/minio:/export \
   minio/minio:RELEASE.2019-12-17T23-16-33Z \
   server /export
```

Create a bucket and you will get the following log:

```
{"version":"1","deploymentid":"edec2a5e-301a-4c33-acfc-218b62d4c785","time":"2020-04-22T11:13:53.103636101Z","api":{"name":"PutBucket","bucket":"test-log","timeToFirstByte":"0s","timeToResponse":"0s"},"remotehost":"10.11.0.1","requestID":"16081FFB5CDB6F61","userAgent":"MinIO (linux; amd64) minio-go/v6.0.52 mc/2020-04-04T05:28:55Z","requestHeader":{"Authorization":"AWS4-HMAC-SHA256 Credential=testtest/20200422/us-east-1/s3/aws4_request, SignedHeaders=host;x-amz-content-sha256;x-amz-date, Signature=358203e1a5d04408ce2709c33f27d2dacd39ed7f094902681d6aa98e1a7037db","Content-Length":"0","User-Agent":"MinIO (linux; amd64) minio-go/v6.0.52 mc/2020-04-04T05:28:55Z","X-Amz-Content-Sha256":"UNSIGNED-PAYLOAD","X-Amz-Date":"20200422T111353Z"},"responseHeader":{"Accept-Ranges":"bytes","Content-Length":"0","Content-Security-Policy":"block-all-mixed-content","ETag":"","Location":"/test-log","Server":"MinIO/RELEASE.2020-04-22T00-11-12Z","Vary":"Origin","X-Amz-Request-Id":"16081FFB5CDB6F61","X-Xss-Protection":"1; mode=block"}}
```

You can then upgrade to the latest release and check that the problem is still present.

**Comment by anthu** [comment]
FYI: Fixed by #9445 and #9496 

Looking forward to the next release as proper Audit logging is critical!
Thanks @vadmeste for this!
